setup.cfg

```diff
@@ -1,12 +1,12 @@
 [flake8]
 max-line-length = 120
-exclude = .venv, migrations, settings, manage.py, shortener, options, frontend
+exclude = .venv, migrations, settings, manage.py, shortener, frontend
```

Comment:
I think we want to remove `shortener` from here too

Reply:
Fixed